_posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Overview and Logs for the Kovri Dev Meeting Held on 2018-06-07
+summary: Discussion of the latest Kovri development updates and the alpha release
+tags: [dev diaries, i2p, crypto]
```

Comment:
Shouldn't we use `kovri` tag too? Is `kovri` used elsewhere?

Reply:
According to other kovri log post, no : `tags: [dev diaries, i2p, crypto]`
i2p is the kovri tag on blog posts.

Reply:
![image](https://user-images.githubusercontent.com/37215310/41140987-977852a8-6af0-11e8-9607-fff30b807e36.png)
![image](https://user-images.githubusercontent.com/37215310/41141006-ac3106e0-6af0-11e8-8eb1-04e78700e3c7.png)